<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectViewWidget</class>
 <widget class="QDockWidget" name="ProjectViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project</string>
  </property>
  <widget class="QWidget" name="widget_contents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QTreeView" name="project_view">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
     <action name="action_show_file_content">
         <property name="text">
             <string>Show file content</string>
         </property>
     </action>
     <action name="action_show_file_viewer">
         <property name="text">
             <string>Show charts</string>
         </property>
     </action>
     <action name="action_read_file">
         <property name="text">
             <string>Read file</string>
         </property>
     </action>
 </widget>
 <resources/>
 <connections/>
</ui>
